app/containers/InterventionDetailsPage/containers/ParticipantInviter/CopyLinkForm.tsx

```diff
@@ -14,13 +14,14 @@ import { SelectOption } from 'components/Select/types';
 import { CopyLinkFormValues } from './types';
 import messages from './messages';
 import { createCopyLinkFormSchema, createInviteUrl } from './utils';
+import { Session } from '../../../../models/Session';
```

Comment:
```suggestion
import { Session } from 'models/Session';
```
and reorder this import